site/tests/app/models/ConfigTester.php

```diff
@@ -75,8 +75,9 @@ private function createConfigFile($extra = array()) {
             "vcs_url" => "",
             "cgi_url" => "http://example.com/cgi-bin",
             "institution_name" => "RPI",
-            "username_change_text" => "Submitty welcomes individuals of all ages, backgrounds, citizenships, disabilities, sex, education, ethnicities, family statuses, genders, gender identities, geographical locations, languages, military experience, political views, races, religions, sexual orientations, socioeconomic statuses, and work experiences. In an effort to create an inclusive environment, you may specify a preferred name to be used instead of what was provided on the registration roster.",
+            "username_change_text" => "Submitty welcomes all students.",
```

Comment:
Why did you change this?

Reply:
It was a super long string that's used just in unit testing and I hate lines that exceed my desired length of 88 characters and really hate lines that exceed 120 characters.

It does not affect the default one that is set on installation.